packages/react/src/Heading/index.ts

```diff
@@ -1,2 +1,3 @@
-export { Heading } from './Heading'
-export type { HeadingProps } from './Heading'
+export type { HeadingLevel, HeadingProps } from './Heading'
```

Comment:
Do we want to expose `HeadingLevel` seperately to the outside world? It feels more like an internal type imo. When would a consumer need it?

Reply:
Accordion does ;) That was one of the things I wanted to reconsider.

Although in general I don’t think component exporting their essential related context is a bad thing.

Reply:
Updated according to our discussion leading to #409.